tests/test_make.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Right this would generally be for testing whether things compile, rather than validating the compiled pdf.